packages/core-app-api/api-report.md

```diff
@@ -409,6 +409,12 @@ export class MicrosoftAuth {
   static create(options: OAuthApiCreateOptions): typeof microsoftAuthApiRef.T;
 }
 
+// @public
+export class MultipleAnalyticsApi implements AnalyticsApi {
+  captureEvent(event: AnalyticsEvent): void;
+  static withApis(actualApis?: AnalyticsApi[]): MultipleAnalyticsApi;
```

Comment:
how do you feel about `fromApis` instead? Had to go and check what we recommend in https://github.com/backstage/backstage/blob/master/STYLE.md#api-design, and it's `create*` or `from*`. I think `with*` is more for fluid-style builder patterns

Reply:
Ah, that definitely makes sense. https://github.com/backstage/backstage/pull/13309/commits/bcb425cfeb756ab451bfc8e6b0d509bc79c869f0